code/create_app.py

```diff
@@ -130,10 +130,14 @@ def conversation_with_data(request: Request, env_helper: EnvHelper):
                         "filter": env_helper.AZURE_SEARCH_FILTER,
                         "in_scope": env_helper.AZURE_SEARCH_ENABLE_IN_DOMAIN,
                         "top_n_documents": env_helper.AZURE_SEARCH_TOP_K,
+                        "embedding_dependency": {
+                            "type": "deployment_name",
+                            "deployment_name": env_helper.AZURE_OPENAI_EMBEDDING_MODEL,
+                        },
                         "query_type": (
-                            "semantic"
+                            "vector_semantic_hybrid"
```

Comment:
Did you consider adding a new env var to allow the user to pick any of the five different search types?

Reply:
I like this!

Reply:
This is a good idea, this will give the user full and transparent control over the query type.

However, so far, Chat with your data has been opinionated (at least via the /api/conversations/custom endpoint) about always using hybrid search (as opposed to text-based-only or vector-only). As of this PR, the /byod endpoint has parity with the /custom endpoint - it always uses hybrid search, and enables semantic search when the user requests it (while still maintaining hybrid search).

I have created #897 to cover the work to allow for all five types of queries to be available as options to the user to choose from.